readme.md

```diff
@@ -132,7 +132,7 @@ It is worth, to begin with taxrules as it is the fastest.
 e.g.
 
 ```
-*/5 * * * * cd [full path to magento directory]/shell && /usr/bin/flock -n /tmp/vsf_index.lock  /usr/bin/php vsf_tools.php --action reindex 
+*/5 * * * * cd [full path to magento directory]/shell && /usr/bin/flock -n /tmp/vsf_index.lock  /usr/bin/php vsf_tools.php -- --action reindex 
 ```
```

Comment:
if we don't use `-f` switch we don't  have to do add `-- --action reindex`. So please remove this last changes from line 135.

Both options will work:
`php -f vsf_tools.php -- --action full_reindex --store STORE_ID --type attributes`
or
`php vsf_tools.php --action full_reindex --store STORE_ID --type attributes`